scripts/patch_release/patch.py

```diff
@@ -0,0 +1,358 @@
+"""
+Patch releases occur when samples need to be retracted due to
+patients withdrawing consent.
+
+Patches should always occur on the latest consortium release for
+the specific public release. Due to the GENIE retraction policy,
+it is best to retract data on the 3rd consortium release of the
+subsequent release series.
+"""
+import argparse
+import os
+import shutil
+import tempfile
+
+import pandas as pd
+import synapseclient
+
+from genie import process_functions
```

Comment:
I think the isort extension/precommit would make a change here to add these imports into a single `from` statement